client/app/scripts/liveblog-edit/directives.js

```diff
@@ -29,8 +29,11 @@ define([
                     angular.extend(vm, {
                         isLoading: true,
                         blogId: $scope.lbPostsBlogId,
-                        allowUnpublish: $scope.lbPostsAllowUnpublish,
-                        allowReordering: $scope.lbPostsAllowReordering,
+                        allowUnpublishing: $scope.lbPostsAllowUnpublishing || false,
```

Comment:
I want boolean for all privileges. If undefined, I assume that the privilege is not given.
I know that undefined can be used as a false value but I prefer an explicit false. Then here, a code reader can understand that we set default values.